address.go

```diff
@@ -55,6 +55,7 @@ type Address struct {
 	Name            string                `json:"name,omitempty"`
 	Company         string                `json:"company,omitempty"`
 	Phone           string                `json:"phone,omitempty"`
+	Message         string                `json:"message,omitempty"`
```

Comment:
Why is `message` being added here? Addresses don't have a message property.

Reply:
Address needs `message` attribute for `create_any_verify` endpoint, C# and Java have `message` attribute in their Address class.

C#: https://github.com/EasyPost/easypost-csharp/blob/master/EasyPost.Net/Address.cs#L28
Java: https://github.com/EasyPost/easypost-java/blob/master/src/main/java/com/easypost/model/Address.java#L24

Reply:
Does it "need" it, or were you just following the C# and Java libs? You'll notice that the response does not contain `message` at all. I believe the `message` key is no longer valid/used. If you check our internal code for the address controller on the API level, there is no mention of a `message` key at all. I assume we previously exposed a bare `message` key, but have since wrapped it up in the `verifications` dictionary.

Reply:
After some internal discussion, the `message` attribute is no longer needed, this also applies to other libraries and we will address them in the future.